sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingJoinSuite.scala

```diff
@@ -996,4 +997,47 @@ class StreamingOuterJoinSuite extends StreamTest with StateStoreMetricsTest with
       )
     }
   }
+
+  test("SPARK-32148 stream-stream join regression on Spark 3.0.0") {
+    val input1 = MemoryStream[(Timestamp, String, String)]
+    val df1 = input1.toDF
+      .selectExpr("_1 as eventTime", "_2 as id", "_3 as comment")
```

Comment:
Any specific reason why not use `select`? I don't see any expression here.

Reply:
I guess it's pretty much simpler and more readable than `select('_1.as("eventTime"), '_2.as("id"), '_3.as("comment"))` (or even with col(...) if ' notation doesn't work for _1, _2, _3).